assets/src/hooks/use_sentry.tsx

```diff
@@ -0,0 +1,13 @@
+import * as Sentry from "@sentry/react";
+
+const useSentry = () => {
+  const sentryDsn = document.getElementById("app")?.dataset.sentry;
+  console.log(sentryDsn);
```

Comment:
make sure to remove this. Also I think the only thing I would add to this just to avoid re-initializing is doing something like....
```js
useEffect(() => {
  if (sentryDsn) {
    Sentry.init( ... )
  }
}, [sentryDsn]);
```